src/main/resources/static/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="uk">
+<head>
+    <meta charset="UTF-8">
+    <title>Обробка квитанцій</title>
+    <style>
+        body {
+            margin: 0;
+            font-family: sans-serif;
+            background-color: #f0f8ff;
+            background-image: repeating-linear-gradient(
+                    -45deg,
+                    rgba(0, 128, 255, 0.05) 0,
+                    rgba(0, 128, 255, 0.05) 100px,
+                    transparent 100px,
+                    transparent 200px
+            ),
+            repeating-linear-gradient(
+                    -45deg,
+                    transparent 0,
+                    transparent 100px,
+                    rgba(0, 128, 255, 0.05) 100px,
+                    rgba(0, 128, 255, 0.05) 200px
+            );
+        }
+
+        .container {
+            text-align: center;
+            padding-top: 100px;
+        }
+
+        .send-button {
+            font-size: 24px;
+            padding: 20px 40px;
+            border-radius: 50px;
+            background-color: #007bff;
+            color: white;
+            border: none;
+            cursor: pointer;
+            margin: 20px;
+            transition: transform 0.5s ease-in-out;
+        }
+
+        .send-button.loading {
+            animation: rotate 1s infinite linear;
+        }
+
+        @keyframes rotate {
+            0% { transform: rotate(0deg); }
+            100% { transform: rotate(360deg); }
+        }
+
+        .success {
+            margin-top: 30px;
+            font-size: 40px;
+            color: white;
+            background-color: #28a745;
+            padding: 20px;
+            border-radius: 20px;
+            display: inline-block;
+        }
+
+        .watermark {
+            position: fixed;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            font-size: 48px;
+            color: rgba(0, 0, 255, 0.05);
+            transform: rotate(-45deg);
+            white-space: nowrap;
+            pointer-events: none;
+            z-index: 0;
+        }
+
+        .content {
+            position: relative;
+            z-index: 1;
+        }
+    </style>
+    <script>
+        function processMeasurements() {
+            const btn = document.getElementById("sendBtn");
+            const resultDiv = document.getElementById("result");
+            resultDiv.innerHTML = "";
+            btn.classList.add("loading");
+
+            fetch("/measurement")
+                .then(response => response.text())
+                .then(result => {
+                    btn.classList.remove("loading");
+                    if (result.includes("All measurements has been processed")) {
+                        resultDiv.innerHTML = "<div class='success'>👍 Успішно!</div>";
+                    } else {
+                        alert(result);
```

Comment:
You can have here also innerHTML to display failure in more attractive way instead of alert